code/modules/mob/living/carbon/human/say.dm

```diff
@@ -10,9 +10,11 @@
 
 /mob/living/carbon/human/treat_message(message)
 	message = dna.species.handle_speech(message,src)
+	var/list/temp_message = splittext(message, " ") //List each word in the message
+	var/speechwords = min(round(temp_message.len), 5)

```

Comment:
Why would you round the length of a list? That is guaranteed to be an unsigned integer.